CONTRIBUTING.md

```diff
@@ -124,8 +124,8 @@ submitting a pull request.
 Update the documentation when creating or modifying features. Test your
 documentation changes for clarity, concision, and correctness, as well as a
 clean documentation build. See our contributors guide for [our style
-guide](https://docs.docker.com/opensource/doc-style) and instructions on [building
-the documentation](https://docs.docker.com/opensource/project/test-and-docs/#build-and-test-the-documentation).
+guide](https://docs.docker.com/contribute/style/grammar/) and instructions on [building
+the documentation](https://github.com/docker/docs/blob/main/CONTRIBUTING.md).
```

Comment:
I think we should avoid links to other github repositories here. I think let's point this to the official docs.

```suggestion
the documentation](https://docs.docker.com/contribute/).
```